app/views/activities/index.html.erb

```diff
@@ -185,7 +185,6 @@
           <% end %>
         <% end %>
       </div>
-    <p class= "end mt-4"><em>Refresh for more activities!</em></p>
```

Comment:
remove the end in css also since it's redundant

Reply:
done